src/main/java/com/iexec/common/replicate/ReplicateStatus.java

```diff
@@ -124,6 +146,10 @@ public static List<ReplicateStatus> getFailureStatuses() {
      * Statuses that should be updated
      * to COMPLETED at the end of a task. 
      */
+    /**
+     * @deprecated This unused method will be cleaned.
+     */
+    @Deprecated(forRemoval = true)
     public static List<ReplicateStatus> getCompletableStatuses() {
```

Comment:
This method is not used in production code but in tests on `iexec-core` side.
Do we intend to move this list explicitly to `iexec-core` tests ?

Reply:
That'd be great, yes.